sdk/core/Azure.Core/src/ResponseError.Serialization.cs

```diff
@@ -140,58 +140,8 @@ private void Write(Utf8JsonWriter writer)
 
         void IJsonModel<ResponseError>.Write(Utf8JsonWriter writer, ModelReaderWriterOptions options)
         {
-            var format = options.Format == "W" ? ((IPersistableModel<ResponseError>)this).GetFormatFromOptions(options) : options.Format;
-            if (format != "J")
-            {
-                throw new FormatException($"The model {nameof(ResponseError)} does not support '{format}' format.");
-            }
-
             writer.WriteStartObject();
```

Comment:
Curious - why leave the write start object here instead of in the write core method? Does it matter that the format is checked after this?

Reply:
this is the pattern we use to allow for inheritance to write their properties and defer to base to write the base properties but the extra { } would be a format failure if they call write directly.